imsv-docs-astro/astro.config.mjs

```diff
@@ -0,0 +1,88 @@
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+import tailwind from "@astrojs/tailwind";
+import markdoc from "@astrojs/markdoc";
+
+// https://astro.build/config
+export default defineConfig({
+  integrations: [
+    starlight({
+      title: 'Immersve Docs',
+      social: {
+        github: 'https://github.com/immersve/'
+      },
+      favicon: '/favicon.ico',
+      customCss: ['./src/styles/tailwind.css'],
+      components: {
+        MarkdownContent: './src/components/MarkdownContent.astro',
+        Pagination: './src/components/FooterNavigation.astro',
+      },
+      sidebar: [
+        { label: 'Use Cases', autogenerate: { directory: 'use-cases' } },
+        {
+          label: 'Guides',
+          items: [
+            { label: 'Introduction', link: 'guides' } ,
+            { label: 'Core Concepts', autogenerate: { directory: 'guides/core-concepts' } },
+            { label: 'Card Issuing Apps', autogenerate: { directory: 'guides/card-issuing-apps' } },
+            { label: 'Funding Protocols', autogenerate: { directory: 'guides/funding-protocols' } },
+            { label: 'Reports', autogenerate: { directory: 'guides/reports' } } ,
+            { label: 'Testing', autogenerate: { directory: 'guides/testing' } } ,
+            { label: 'Authentication', link: 'guides/authentication' } ,
+            { label: 'Webhooks', link: 'guides/webhooks' } ,
+          ]
+        },
+        {
+          label: 'API Reference',
+          link: '/api-reference/',
+        },
+      ],
+    }),
+    tailwind({
+      applyBaseStyles: false
+    }),
+    markdoc(),
+  ],
+
+  redirects: {
+    '/api-reference/accounts-custodial': '/guides/custodial-wallets',
+    '/api-reference/activate-a-card-for-physical-cards-only': '/api-reference/card',
+    '/api-reference/assets': '/api-reference/asset-activities',
+    '/api-reference/block-a-card': '/api-reference/card',
+    '/api-reference/cancel-a-card-asynchronously': '/api-reference/cancel-a-card',
+    '/api-reference/cancel-an-account': '/use-cases/custodial-wallets',
+    '/api-reference/cards-custodial': '/api-reference/card',
+    '/api-reference/claim-a-funding-source-for-an-account': '/api-reference/create-a-funding-source-for-an-account',
+    '/api-reference/close-a-card': '/api-reference/cancel-a-card',
+    '/api-reference/create-a-new-card': '/api-reference/order-card',
```

Comment:
Question: do we plan to keep deprecated endpoints in the docs for reference?

Reply:
@catluin I think it depends. These ones were renamed before anyone integrated so we don't need to keep the docs. The redirect is there so links from Slack or Notion etc are preserved.